### PR TITLE
Corelight Zeek Modeling & Parsing

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CorelightZeek/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Corelight Zeek
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Corelight Sensor to forward Zeek Syslog messages.
+
+1. Open Corelight Sensor UI, at the left bar choose "Configuration".
+2. Open the "Export" tab and scroll down to "Export to Syslog";
+    * Under "Syslog Server"- Set you XSIAM IP and Port numbers.
```
Comment (ShirleyDenkberg) [inline Packs/CorelightZeek/README.md:8]:
```suggestion
    * Under "Syslog Server"- Set your XSIAM IP and Port numbers.
```

File: Packs/CorelightZeek/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Corelight Zeek
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Corelight Sensor to forward Zeek Syslog messages.
+
+1. Open Corelight Sensor UI, at the left bar choose "Configuration".
+2. Open the "Export" tab and scroll down to "Export to Syslog";
```
Comment (ShirleyDenkberg) [inline Packs/CorelightZeek/README.md:7]:
```suggestion
2. Open the **Export** tab and scroll down to **Export to Syslog**.
```

File: Packs/CorelightZeek/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Corelight Zeek
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Corelight Sensor to forward Zeek Syslog messages.
+
+1. Open Corelight Sensor UI, at the left bar choose "Configuration".
```
Comment (ShirleyDenkberg) [inline Packs/CorelightZeek/README.md:6]:
```suggestion
1. Open the Corelight Sensor UI, and on the left menu bar click **Configuration**.
```

File: Packs/CorelightZeek/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+2. Open the "Export" tab and scroll down to "Export to Syslog";
+    * Under "Syslog Server"- Set you XSIAM IP and Port numbers.
+    * Under "Syslog Format"- Pick the "Default" option.
+    * Under "Syslog Facility"- Set your Syslog Facility, E.g. Local0.
```
Comment (ShirleyDenkberg) [inline Packs/CorelightZeek/README.md:10]:
```suggestion
    * Under "Syslog Facility"- Set your Syslog Facility, e.g., Local0.
```

File: Packs/CorelightZeek/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    * Under "Syslog Severity"- Select the severity for the logs you will sent, E.g. Info.
```
Comment (ShirleyDenkberg) [inline Packs/CorelightZeek/README.md:11]:
```suggestion
    * Under "Syslog Severity"- Select the severity for the logs you will sent, e.g., Info.
```

File: Packs/CorelightZeek/README.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+# Corelight Zeek
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+You need to configure Corelight Sensor to forward Zeek Syslog messages.
+
+1. Open Corelight Sensor UI, at the left bar choose "Configuration".
+2. Open the "Export" tab and scroll down to "Export to Syslog";
+    * Under "Syslog Server"- Set you XSIAM IP and Port numbers.
+    * Under "Syslog Format"- Pick the "Default" option.
```
Comment (ShirleyDenkberg) [inline Packs/CorelightZeek/README.md:9]:
```suggestion
    * Under **Syslog Format**- Pick the "Default" option.
```

File: Packs/CorelightZeek/README.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    * Under "Syslog Facility"- Set your Syslog Facility, E.g. Local0.
+    * Under "Syslog Severity"- Select the severity for the logs you will sent, E.g. Info.
+3. Open the "Maintain" tab.
```
Comment (ShirleyDenkberg) [inline Packs/CorelightZeek/README.md:12]:
```suggestion
3. Open the **Maintain** tab.
```

File: Packs/CorelightZeek/README.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+4. Under "Performance Monitoring", enable the "Reporting to Syslog (UDP-Only)" option and set a time interval (E.g. 1min).
```
Comment (ShirleyDenkberg) [inline Packs/CorelightZeek/README.md:13]:
```suggestion
4. Under **Performance Monitoring**, enable the **Reporting to Syslog (UDP-Only)** option and set a time interval (e.g., 1min).
```

File: Packs/CorelightZeek/README.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+5. Under "Performance Monitoring", enable the "Reporting Through Zeek Log Streams" option and set a time interval (E.g. 1min).
```
Comment (ShirleyDenkberg) [inline Packs/CorelightZeek/README.md:14]:
```suggestion
5. Under **Performance Monitoring**, enable the **Reporting Through Zeek Log Streams** option and set a time interval (e.g., 1min).
```

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**